src/js/other-websites/add_input_icon.js

```diff
@@ -47,16 +67,16 @@ function buildInpageIframe(opts) {
   );
   const iframe = document.createElement("iframe");
   iframe.src = browser.runtime.getURL("inpage-panel.html");
-  iframe.width = 300;
-  iframe.height = 205;
+  iframe.width = 320;
+  iframe.height = 300;
```

Comment:
🪄  Magic Number™ derived from the Figma file. However, this is just the starting instance of the iframe. After it's built out, it resizes itself based on the inner contents.

Reply:
Comment added in 910f7c0ee528233571fd5878b68f59c3a9b067d6